src/InverseSolve/models.jl

```diff
@@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
 	return f(m),sdiag(df(m))
 end
 
+function identityMod(m)
+	return m,1.0
```

Comment:
Just pushed this.

On Mon, Apr 18, 2016 at 4:32 PM, Lars Ruthotto <notifications@github.com>
wrote:

> In src/InverseSolve/models.jl
> <https://github.com/JuliaInv/jInv.jl/pull/3#discussion_r60152492>:
>
> > @@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
> >  	return f(m),sdiag(df(m))
> >  end
> >
> > +function identityMod(m)
> > +	return m,1.0
>
> small thing. How about using UniformScaling(1.0) instead of 1.0? Thereby
> the return should be equivalent to an identity matrix.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaInv/jInv.jl/pull/3/files/385c3c914b2994f15ed3bc5315e2cf92332c5045#r60152492>
>